less/about/preferences.less

```diff
@@ -1001,43 +1012,47 @@ table.sortableTable {
     .paymentsSwitches {
       display: flex;
 
-      .enablePaymentsSwitch, .autoSuggestSwitch {
+      .enablePaymentsSwitch,
+      .autoSuggestSwitch {
         display: flex;
         align-items: center;
         width: @walletBarTableData;
 
-        span, .moreInfoBtn {
-          font-weight: bold;
-          color: @gray;
+        .settingItem .switchControl {
+          padding-top: 0;
+          padding-bottom: 0;
         }
 
+        span,
         .moreInfoBtn {
-          margin-left: 7px;
-          cursor: pointer;
-          text-decoration: none;
+          font-weight: bold;
+          font-size: @fontSize;
+          color: @gray;
         }
 
         label {
           font-weight: bold;
           color: @braveOrange;
         }
       }
-    }
 
-    .autoSuggestSwitch {
-      margin-right: 25px;
-      width: 220px;
-    }
+      .autoSuggestSwitch {
+        position: relative;
+        left: -5px; // .prefTabContainer .switchControl
 
-    .advancedSettings {
-      min-width: 235px; // .walletBar td min-width
+        .moreInfoBtn {
+          margin-left: 7px;
+          cursor: pointer;
+          text-decoration: none;
+        }
+      }
     }
   }
 
   .paymentsMessage {
     background-color: @lightGray;
     border-radius: @borderRadiusUIbox;
-    margin: @walletBarMargin;
+    margin-top: @walletBarMargin;
```

Comment:
This fixes the regression I introduced with the former commit of the PR.

Test Plan: 
1. Disable payments
2. Make sure the sidebar is not wrapped